srcpkgs/android-studio/template

```diff
@@ -1,83 +1,71 @@
 # Template file for 'android-studio'
 pkgname=android-studio
-version=4.1.3
+version=2022.3.1.18
 revision=1
-# _studio_build and _studio_rev are for downloading the zip from dl.google.com
-# https://developer.android.com/studio/#resources as of 2018-07-12
-_studio_build=201.7199119
-_studio_rev=0
-archs="x86_64 i686"
+archs="x86_64"
 hostmakedepends="tar"
-depends="virtual?libGL"
 short_desc="Official Android IDE"
-maintainer="Jordyn Carattini <onlinecloud1@gmail.com>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
Why are you orphaning a package that you don't maintain?

Reply:
package have not updated from long time

Reply:
should I revert it ?

Reply:
Looks like the maintainer hasn't contributed to Void in several years, so we can keep the orphan change.